scripts/bump-version.mjs

```diff
@@ -74,12 +75,19 @@ try {
 }
 
 // Select which package to deploy ('field-plugin' | 'cli')
-const { packageFolder } = await prompts({
-  type: 'select',
-  name: 'packageFolder',
-  message: 'What to deploy?',
-  choices: PACKAGE_FOLDERS,
-})
+const { packageFolder } = await prompts(
```

Comment:
This file is irrelevant in this PR, but added this changes because they're aligned with the changes above.

Reply:
Did you mean to use `betterPrompts` in this file?